exaudfclient/base/exaudflib/exaudflib.h

```diff
@@ -21,7 +21,7 @@ bool send_run(zmq::socket_t &socket);
 void send_undefined_call(zmq::socket_t &socket, const std::string& fn);
 bool send_done(zmq::socket_t &socket);
 void send_finished(zmq::socket_t &socket);
-
+bool send_return(zmq::socket_t &socket, const char* result);
 
 
```

Comment:
The PythonVM and JavaVM further below we could also extract, similar to the RVM. The SwigVM class is not needed by swig, so maybe we could extract that as well. The other stuff, we might not be able to extract, because swig might need it.

Reply:
Done